surveys/2021-annual-survey/questions.md

```diff
@@ -131,6 +131,22 @@ Select 0-4 (0 = I can't write or read Rust, 1 = I can write simple exercises in
 >
 > Useful for cohort analysis, i.e., for other questions we can query if answers are significantly different for beginners vs advanced users.
 
+### How helpful/effective were the following activities for learning Rust?
+
+Activities:
+
+- Reading books or other written material geared towards learning Rust
+- Watching videos, streams, etc. geared towards learning Rust
+- Doing Rust coding exercises or challenges created to help learn Rust
+- Building a non-trivial project in Rust
```

Comment:
```suggestion
- Building a non-trivial project in Rust or contributing to an open source project
```

Reply:
Perhaps add 'Attending an organized training session or course, virtually or in person'